test/wdio/tsconfig-global-script.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "./tsconfig-stencil.json",
+  "include": [
+    "global-script"
+  ],
+  "exclude": [
+    "global-script/*.test.tsx"
+  ]
+}
```

Comment:
A minimal tsconfig just for the global scripts - note that this is the one which is specified in the `global-script`-specific Stencil configuration file

Reply:
For my own understanding, why not copy the existing one that's pre-configured/existing? There are some niceties like `alwaysStrict` in there already, why not just take what we have and iterate on it at a later point?

Reply:
I'm not sure I follow - this one inherits from `tsconfig-stenci.json` so it will get all the same compiler options